packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -0,0 +1,136 @@
+import MyAlgoConnect, { Accounts, Address, SignedTx } from "@randlabs/myalgo-connect";
+import algosdk, { Transaction } from "algosdk";
+
+import { mkTxParams } from "..";
+import { ExecParams, TransactionInGroup } from "../types";
+import { algoexplorerAlgod } from "./api";
+import { mkTransaction } from "./txn";
+
+const CONFIRMED_ROUND = "confirmed-round";
+const LAST_ROUND = "last-round";
+
+export class MyAlgoWalletSession {
+	readonly connector: MyAlgoConnect;
+	private readonly algodClient: algosdk.Algodv2;
+	accounts: Accounts[] = [];
+	addresses: Address[] = [];
+
```

Comment:
To avoid importing `@randlabs/myalgo-connect` create / copy interface types here and you them in the code instead of the types imported from `@randlabs/myalgo-connect`. Then in constructor only dynamically import `MyAlgoConnect` to initialize the connection object.

````
type Connector interface {
   ...   
}
```